boards/DIM/Inc/App/App_RegenPaddle.h

```diff
@@ -38,6 +39,8 @@ ExitCode App_RegenPaddle_GetRawPaddlePosition(
  * Get the mapped paddle position for the given regen paddle, where 0 means no
  * regen and 100 means maximum regen
  * @param regen_paddle The regen paddle to get mapped paddle position for
+ * @param returned_mapped_paddle_position Buffer to hold the mapped paddle
+ *                                        position
```

Comment:
When I think of a `buffer` I generally think of 1 or more values, but this just looks like a single value? Perhaps `This will be set to the mapped paddle position` might be a bit more explicit?

Reply:
Fine with me :+1: